wp-cache-phase2.php

```diff
@@ -180,17 +180,15 @@ function wp_cache_serve_cache_file() {
 
 			// don't try to match modified dates if using dynamic code.
 			if ( $wp_cache_mfunc_enabled == 0 && $wp_supercache_304 ) {
-				if ( function_exists( 'apache_request_headers' ) ) {
-					$request = apache_request_headers();
-					$remote_mod_time = ( isset ( $request[ 'If-Modified-Since' ] ) ) ? $request[ 'If-Modified-Since' ] : null;
-				} else {
-					if ( isset( $_SERVER[ 'HTTP_IF_MODIFIED_SINCE' ] ) )
-						$remote_mod_time = $_SERVER[ 'HTTP_IF_MODIFIED_SINCE' ];
-					else
-						$remote_mod_time = null;
+				$headers         = apache_request_headers();
```

Comment:
Is it safe to presume that apache_request_headers() is available? While it's unlikely, there may be users running very old versions of PHP in unusual setups.

Reply:
We have fallback for this function into `wp-cache-phase2.php` (I've moved it from `wp-cache.php` to end of `wp-cache-phase2.php` where is optimized version). In this case, I think that's safe to we presume that this function exists (code is more readable and it's little performance improvement).